docs/11-circleci/01-getting-started.mdx

```diff
@@ -0,0 +1,149 @@
+---
+toc_max_heading_level: 4
+---
+
+# Getting started
+
+This guide will help you test and build your Unity project using our
+[Unity Orb](https://circleci.com/developer/orbs/orb/game-ci/unity).
+
+## Overall steps
+
+1. Understand how CircleCI works.
+2. Set up a workflow for your project.
+3. Download the artifacts from your first build.
+
+## CircleCI
```

Comment:
I kinda like how the [GitLab section](https://game.ci/docs/gitlab/getting-started#setting-up-gitlab-ci-for-your-unity-project) had the option for the reader to use a sample project if they didn't already have their own. I wonder if it would make sense to do the same here as well. Perhaps with https://github.com/CircleCI-Public/Unity2D-Demo-Game-CI-CD?

Reply:
That's a pretty good idea! I need to update the demo project before referencing it, so I might not include it in this PR. But I'll 100% add it.